app/Http/Controllers/Tenancy/TenantController.php

```diff
@@ -37,9 +38,20 @@ public function viewTenant()
         $response = json_decode($responseBody, true);
 
         $de = collect($response['message'])->paginate(5);
-        // $de = $data->all();
+        $cloudButton = StatusSetting::value('cloud_button');
 
-        return view('themes.default1.tenant.index', compact('cloud', 'de'));
+        return view('themes.default1.tenant.index', compact('cloud', 'de', 'cloudButton'));
+    }
+
+    public function enableCloud(Request $request)
+    {
+        try {
+            $request->input('debug') == 'true' ? StatusSetting::where('id', '1')->update(['cloud_button' => '1']) : StatusSetting::where('id', '1')->update(['cloud_button' => '0']);
+
```

Comment:
**StatusSetting::first()->update(['cloud_button' => (int) $request->input('debug')]);**

Above can be replaced with this, i think Can you check please